### PR TITLE
Icon Request: Skynet

### PR DESCRIPTION
**Pull request by kwypchlo**
skynet

**Issue:** https://github.com/simple-icons/simple-icons/issues/6118
**Alexa rank:** 157,547
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
There is an existing pull request https://github.com/simple-icons/simple-icons/pull/6139 for this icon but it's been inactive for a while so I decided to prepare one myself

closes https://github.com/simple-icons/simple-icons/issues/6118

### CODE REVIEW

**Comment by kwypchlo** [comment]
@service-paradis is there anything else that should be done before this icon gets merged? Skynet is preparing for new product launch and badge from shields.io is a part of the marketing effort, it would be great if we could get this merged and use the shorthand for the icon on it

**Comment by service-paradis** [comment]
Nothing from you for now. The missing part is the review from one of the project maintainers. We've all been quite busy for the past weeks, but this PR turns will come. During the review process, we'll let you know if some changes are required. If not, your PR will be merged.

**Review by service-paradis — APPROVED** [review]
Good job @kwypchlo, it is a perfect match 💯 Thank you for your contribution!

**Pull request merged**
